src/sonic-config-engine/portconfig.py

```diff
@@ -376,7 +378,8 @@ def get_config(self):
                     'alias': self._breakout_capabilities[alias_id],
                     'lanes': ','.join(lanes),
                     'speed': str(entry.default_speed),
-                    'index': self._indexes[lane_id]
+                    'index': self._indexes[lane_id],
+                    'subport': "0" if total_num_ports == 1 else str(alias_id + 1)
```

Comment:
Will this flow be called for both Static and Dynamic breakouts? Like during load_minigraph etc?

Reply:
Yes this is called both on config_db.json creation + dynamic breakouts.

For example
```
Ports to be deleted :
 {
    "Ethernet0": "400000"
}
Ports to be added :
 {
    "Ethernet0": "100000",
    "Ethernet2": "100000",
    "Ethernet4": "100000",
    "Ethernet6": "100000"
}
Breakout process got successfully completed.
Please note loaded setting will be lost after system reboot. To preserve setting, run `config save`.
root@bthd104:/# sonic-db-cli APPL_DB hgetall PORT_TABLE:Ethernet4
{'alias': 'Ethernet1/5', 'index': '1', 'lanes': '5,6', 'speed': '100000', 'subport': '3', 'mtu': '9100', 'admin_status': 'down'}
root@bthd104:/# sonic-db-cli APPL_DB hgetall PORT_TABLE:Ethernet0
{'alias': 'Ethernet1/1', 'index': '1', 'lanes': '1,2', 'speed': '100000', 'subport': '1', 'mtu': '9100', 'oper_status': 'down', 'admin_status': 'down'}
root@bthd104:/#
```